t/proto_template/protos/v1/person_service/person.proto

```diff
@@ -2,7 +2,7 @@ syntax = "proto3";
 
 package person.v1;
 
-option go_package = "github.com/github_org_name/github_repo_name/v1/person;personpb";
+option go_package = "v1/person;personpb";
```

Comment:
Im removing the prefix here because this causes a redundancy when importing the proto from the client application